### PR TITLE
Add ami-cloudformation-parameter deployment

### PR DESCRIPTION
**Pull request by TBonnin**

### CODE REVIEW

File: riff-raff.yaml [hunk 1]
```diff
@@ -4,4 +4,14 @@ deployments:
...
+    dependencies:
+      - update-ami
+  update-ami:
+    type: ami-cloudformation-parameter
+    app: prism
+    parameters:
+      amiParameter: MachineImageAMI
+      amiTags:
+        Recipe: xenial-java8-deploy-infrastructure
+        AmigoStage: PROD
```
Comment (sihil) [inline riff-raff.yaml]: I suggest that it would be good practice to put `BuiltBy: amigo` on this list to be confident. What do you think?

Comment (TBonnin) [inline riff-raff.yaml]: @sihil Isn't the `AmigoStage` tag kinda plays the same role of making sure it has been built by amigo?

Comment (TBonnin) [inline riff-raff.yaml]: @sihil added the `BuiltBy: amigo`

Comment (sihil) [inline riff-raff.yaml]: Sorry for not replying to this. For the record we discussed this and my argument was that `AmigoStage` is a proxy for this, but it's not the main point of the tag. In hindsight this should be `Stage` and then we should use `BuiltBy` to determine what platform built it.

I guess the other side of the argument would be that we don't care what builds it, we just want the latest version.

**Pull request merged**